examples/complete/main.tf

```diff
@@ -1 +1,28 @@
-#(empty)
+module "complete_example" {
```

Comment:
In the complete example, is it possible to create a load balancer using the elb module to showcase how to use this module for REGIONAL scope?